_posts/2018-08-30-numfocus-workshop.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: NumFOCUS Sponsored Workshop and Hackathon
+---
+
+We are happy to announce that thanks to a [NumFOCUS development grant][numfocus]
+we will be hosting a 2 day workshop and hackathon aimed at introducing researchers to MDAnalysis.
+The event will be free to attend and will be held at Northwestern University, Evanston IL on the 12th and 13th of November 2018.
+In addition, there are travel grants available to attendees aimed at promoting diversity in STEM.
+
+The first day of the workshop will focus on covering the basics of the library and how to apply
+this to your own research.
+The second day of the workshop will be designed to help attendees
```

Comment:
The second day of the workshop will introduce git and GitHub and help you start contributing to MDAnalysis with your own analysis code.